index.bs

```diff
@@ -717,11 +695,9 @@ partial interface MLContext {
             1. Throw an {{OperationError}} {{DOMException}} and stop.
         1. Else:
             1. Let |outputTensor| be the output tensor returned by |graph|.{{MLGraph/[[implementation]]}}.
-            1. If the kind of |value| is not compatible with the value type of |outputTensor|, then throw a {{DataError}} {{DOMException}} and stop.
-            1. Let |outputSize| be 1.
-            1. For each |dimension| of dimensions of |outputTensor|:
-                1. Set |outputSize| to the product of |outputSize| and |dimension|.
-            1. If |outputSize| is greater than the length of |value|, then:
+            1. Let |outputDesc| be |graph|.{{MLGraph/[[outputDescriptors]]}}[|key|].
+            1. If the data type of |outputTensor| is not compatible with |outputDesc|.{{MLOperandDescriptor/type}}, then throw a {{DataError}} {{DOMException}} and stop.
```

Comment:
nit: ideally "compatible" should link to an algorithm explaining what's compatible.

Reply:
Agreed. Thanks for pointing it out. Actually this PR inherits this issue. Could we file a corresponding issue in github and fix it in a separate PR?

Reply:
@wacky6 , I proposed a fix in the latest commit that validates the data type against the element type of TypedArray (by referring the corresponding spec definition). Please take another look whether it looks good. Thanks.